sklearn/discriminant_analysis.py

```diff
@@ -621,7 +630,7 @@ def fit(self, X, y):
         """Fit the model according to the given training data and parameters.
 
             .. versionchanged:: 0.19
-               *store_covariance* has been moved to main constructor.
+               *store_covariances* has been moved to main constructor.
```

Comment:
Or "store_covariance has been moved to the main constructor as store_covaraiances"